python/ray/node.py

```diff
@@ -349,25 +349,20 @@ def _make_inc_temp(self, suffix="", prefix="", directory_name=None):
         raise FileExistsError(errno.EEXIST,
                               "No usable temporary filename found")
 
-    def new_log_files(self, name, redirect_output=True):
+    def new_log_files(self, name):
         """Generate partially randomized filenames for log files.
 
         Args:
             name (str): descriptive string for this log file.
-            redirect_output (bool): True if files should be generated for
-                logging stdout and stderr and false if stdout and stderr
-                should not be redirected.
-                If it is None, it will use the "redirect_output" Ray parameter.
 
         Returns:
-            If redirect_output is true, this will return a tuple of two
-                file handles. The first is for redirecting stdout and the
-                second is for redirecting stderr.
-                If redirect_output is false, this will return a tuple
-                of two None objects.
+            A tuple of two file handles for redirecting (stdout, stderr).
         """
+        redirect_output = self._ray_params.redirect_output
+
         if redirect_output is None:
-            redirect_output = self._ray_params.redirect_output
+            redirect_output = os.getenv("GLOG_logtostderr") != "1"
```

Comment:
Can you comment the reasoning for this change (or at least comment the link to this PR)? It'll be hard to understand why this code is here without any comment.

Reply:
@rkooo567 Done, thanks!